packages/neo-one-server-plugin-wallet/src/__e2e__/bootstrap.test.js

```diff
@@ -0,0 +1,52 @@
+/* @flow */
+import { addressToScriptHash } from '@neo-one/client';
+
+describe('bootstrap', () => {
+  test('bootstrap - default', async () => {
+    const numWallets = 10;
+
+    await one.execute('create network boottest');
+    await new Promise(resolve => setTimeout(() => resolve(), 10000));
+
+    await one.execute('bootstrap --network boottest');
+    await new Promise(resolve => setTimeout(() => resolve(), 10000));
```

Comment:
I think you can remove this `setTimeout` since the command should only complete once it's fully done. Even the `create network` one might not be necessary anymore because we wait for the node to start up before finishing the command. In the network e2e IIRC I added it before we were waiting for the node to start up.